### PR TITLE
implement to_ieee754_32 scalar function for presto

### PR DESCRIPTION
**Pull request by ashokku2022**
resolve https://github.com/facebookincubator/velox/issues/4425

### CODE REVIEW

**Comment by ashokku2022** [comment]
```
I20230406 03:44:51.494438 14964343 ExpressionFuzzer.cpp:974] Format: functionName numTimesSelected proportionOfTimesSelected numProcessedRows
I20230406 03:44:51.494453 14964343 ExpressionFuzzer.cpp:978] to_ieee754_32 6040 100.00% 193876
I20230406 03:44:51.494472 14964343 ExpressionFuzzer.cpp:984] ==============================> Bottom 1 by number of rows processed
I20230406 03:44:51.494486 14964343 ExpressionFuzzer.cpp:986] Format: functionName numTimesSelected proportionOfTimesSelected numProcessedRows
I20230406 03:44:51.494500 14964343 ExpressionFuzzer.cpp:991] to_ieee754_32 6040 100.00% 193876
I20230406 03:44:51.494516 14964343 ExpressionFuzzer.cpp:1004] ==============================> All stats sorted by number of times the function was chosen
I20230406 03:44:51.494531 14964343 ExpressionFuzzer.cpp:1006] Format: functionName numTimesSelected proportionOfTimesSelected numProcessedRows
I20230406 03:44:51.494545 14964343 ExpressionFuzzer.cpp:1010] to_ieee754_32 6040 100.00% 193876
```

**Comment by ashokku2022** [comment]
@aditi-pandit if you could review it when you get time that would be a great help, thanks!

**Comment by ashokku2022** [comment]
@aditi-pandit any feedback would be highly appreciated!

File: velox/functions/prestosql/tests/StringFunctionsTest.cpp [hunk 1]
```diff
@@ -1698,6 +1698,21 @@ TEST_F(StringFunctionsTest, toBase64Url) {
   EXPECT_EQ("_0-_UA==", toBase64Url(fromHex("FF4FBF50")));
 }
 
+TEST_F(StringFunctionsTest, toIEEE754_32) {
+  const auto toIEEE754_32 = [&](std::optional<float> value) {
+    return evaluateOnce<std::string, float>("to_ieee754_32(c0)", value);
+  };
+
+  EXPECT_EQ(std::nullopt, toIEEE754_32(std::nullopt));
```
Comment (aditi-pandit) [inline velox/functions/prestosql/tests/StringFunctionsTest.cpp:1706]: Please can you add more tests for very large and very small values.

**Review by aditi-pandit — COMMENTED** [review]
Hi, Thanks for your PR. 

Just fyi...We are looking at a refactoring of StringImpl.h in https://github.com/facebookincubator/velox/pull/4535. This would affect the changes in this PR.

File: velox/functions/prestosql/tests/StringFunctionsTest.cpp (hunk 1 above)
Comment (ashokku2022) [reply velox/functions/prestosql/tests/StringFunctionsTest.cpp:1706]: Absolutely, let me add more tests. Also, thanks for the heads-up on the upcoming split. I'll wait for split PR to get merged and then make changes in PR accordingly.

Comment (ashokku2022) [reply velox/functions/prestosql/tests/StringFunctionsTest.cpp:1706]: @aditi-pandit I have added the smallest and largest float number testcases.

**Comment by ashokku2022** [comment]
branch got corrupted after refactoring therefore closing it and have open a new PR for the same function. Thanks!

**Pull request closed without merging**